wgsl/index.bs

```diff
@@ -928,6 +899,32 @@ param_list
   </xmp>
 </div>
 
+### Builtin Functions ### {#builtin-functions}
+
+<table class='data'>
+  <thead>
+    <tr><td>Builtins<td>SPIR-V
+  </thead>
+  <tr><td>dpdx(IDENT) -&gt; float<td>OpDPdx
+  <tr><td>dpdx_coarse(IDENT) -&gt; float<td>OpDPdxCoarse
+  <tr><td>dpdx_fine(IDENT) -&gt; float<td>OpDPdxFine
+  <tr><td>dpdy(IDENT) -&gt; float<td>OpDPdy
+  <tr><td>dpdy_coarse(IDENT) -&gt; float<td>OpDPdyCoarse
```

Comment:
There is a slight change of syntax here from the original `dpdy<coarse>` version to the `_`'d version. I'm ok with this, but we should make sure we run it past the group.

Reply:
Yep, I forgot the "for meeting" tag!